easywsdl/2.1.0.wso2v1/pom.xml

```diff
@@ -0,0 +1,124 @@
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.orbit.org.ow2.easywsdl</groupId>
+    <artifactId>easywsdl-all</artifactId>
+    <packaging>bundle</packaging>
+    <name>WSO2 Carbon Orbit - Easywsdl</name>
+    <version>2.1.0.wso2v1</version>
+    <description>
+        This bundle will export packages from Easywsdl
+    </description>
+    <url>http://wso2.org</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.ow2.easywsdl</groupId>
+            <artifactId>easywsdl-wsdl</artifactId>
+            <version>${easywsdl.wsdl.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.ow2.easywsdl</groupId>
+            <artifactId>easywsdl-schema</artifactId>
+            <version>${easywsdl.schema.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind.wso2</groupId>
+            <artifactId>jaxb</artifactId>
+            <version>${com.sun.xml.bind.wso2.jaxb.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jvnet.jaxb2_commons</groupId>
+            <artifactId>runtime</artifactId>
+            <version>${org.jvnet.jaxb2_commons.runtime.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.ow2.easywsdl.wsdl.*;version="${easywsdl.wsdl.version}",
+                            org.ow2.easywsdl.schema.*;version="${easywsdl.schema.version}"
+                        </Export-Package>
+                        <Import-Package>
+                            org.apache.commons.lang.*;version="${commons.lang.wso2.version}"
+                        </Import-Package>
+                        <Private-Package>
+                        </Private-Package>
+                        <Embed-Dependency>
+                            runtime;scope=compile|runtime;,
+                            jaxb;scope=compile|runtime;
```

Comment:
Hi Kalpa,

'org.jvnet.jaxb2_commons'.runtime' jar is only used in this component.
Since there is an another jaxb bundle already in wso2, exposing both can cause platform wise issues.So, with the fact that this is only used in easywsdl we can keep it embedded.

'com.sun.xml.bind.wso2.jaxb' bundle is already available as an osgi bundle. But for historical reasons it's been removed from bundles.info in G-Reg. Thus if we enable that bundle the G-Reg server causes exceptions in startup. Thus, we need to pack this bundle inside of the easywsdl orbit inorder to use in runtime.

Thanks